doc/Programming-guide/Your-first-Kaa-application/index.md

```diff
@@ -438,15 +441,22 @@ int main(void)
     // as soon as the Kaa client is started.
     self.kaaClient = [KaaClientFactory clientWithStateDelegate:self];
     
+    // Configure log upload strategy in order to upload logs right after adding a new one, set maximum of parallel log uploads to 1.
+    RecordCountLogUploadStrategy *logUploadStrategy = [[RecordCountLogUploadStrategy alloc] initWithCountThreshold:1];
```

Comment:
// Configure log uploading strategy so that logs are uploaded upon adding a new log. Set maximum parallel log uploads to 1.